src/node/NodeCreateTransaction.js

```diff
@@ -0,0 +1,441 @@
+        this._accountId =
+            accountId instanceof AccountId
+                ? accountId
+                : AccountId.fromString(accountId);
+
+        return this;
+    }
+
+    /**
+     * @description Get node account identifier.
+     * @returns {?AccountId}
+     */
+    get getAccountId() {
+        return this._accountId;
+    }
+
+    /**
+     * @param {string} description
+     * @description Set description of the node.
+     * @returns {NodeCreateTransaction}
+     */
+    setDescription(description) {
+        if (description.length > DESCRIPTION_MAX_LENGTH) {
+            throw new Error("Description must be at most 100 characters.");
+        }
+        this._description = description;
+
+        return this;
+    }
+
+    /**
+     * @description Get description of the node.
+     * @returns {?string}
+     */
+    get getDescription() {
+        return this._description;
+    }
+
+    /**
+     * @param {ServiceEndpoint[]} gossipEndpoints
+     * @description Set list of service endpoints for gossip.
+     * @returns {NodeCreateTransaction}
+     */
+    setGossipEndpoints(gossipEndpoints) {
+        if (gossipEndpoints.length == 0) {
+            throw new Error("GossipEndpoints list must not be empty.");
+        }
+
+        if (gossipEndpoints.length > GOSSIP_ENDPOINTS_MAX_LENGTH) {
+            throw new Error(
+                `GossipEndpoints list must not contain more than ${GOSSIP_ENDPOINTS_MAX_LENGTH} entries.`,
+            );
+        }
+
+        this._gossipEndpoints = [...gossipEndpoints];
+
+        return this;
+    }
+
+    /**
+     * @description Get list of service endpoints for gossip.
+     * @returns {?Array<ServiceEndpoint>}
+     */
+    get getGossipEndpoints() {
+        return this._gossipEndpoints;
+    }
+
+    /**
+     * @param {ServiceEndpoint} endpoint
+     * @description Add an endpoint to the list of service endpoints for gossip.
+     * @returns {NodeCreateTransaction}
+     */
+    addGossipEndpoint(endpoint) {
+        this._gossipEndpoints.push(endpoint);
+        return this;
+    }
+
+    /**
+     * @param {ServiceEndpoint[]} serviceEndpoints
+     * @description Set list of service endpoints for gRPC calls.
+     * @returns {NodeCreateTransaction}
+     */
+    setServiceEndpoints(serviceEndpoints) {
+        if (serviceEndpoints.length == 0) {
+            throw new Error("ServiceEndpoints list must not be empty.");
+        }
+
+        if (serviceEndpoints.length > SERVICE_ENDPOINTS_MAX_LENGTH) {
+            throw new Error(
+                `ServiceEndpoints list must not contain more than ${SERVICE_ENDPOINTS_MAX_LENGTH} entries.`,
+            );
+        }
+
+        this._serviceEndpoints = [...serviceEndpoints];
+
+        return this;
+    }
+
+    /**
+     * @description Get list of service endpoints for gRPC calls.
+     * @returns {?Array<ServiceEndpoint>}
+     */
+    get getServiceEndpoints() {
+        return this._gossipEndpoints;
```

Comment:
I think you meant `_serviceEndpoints` instead of `_gossipEndpoints` as the method is `getServiceEndpoints()` and the method is for gRPC calls not for gossip
```suggestion
        return this._serviceEndpoints;
```